woocommerce/google-listings-and-ads
# Update DEWP-related tools and docs

### Changes proposed in this Pull Request:

- [Update outdated:dewp scripts](https://github.com/woocommerce/google-listings-and-ads/commit/c31bfd2de26c941a40fa00c768d296d4660b25f1) 
   to show only top-level dependencies.
   In npm 7, the behavior of `npm outdated package-name` changed to show versions for *all* its occurrences within the dependency tree.
- [Describe the use of `dewped` tool](https://github.com/woocommerce/google-listings-and-ads/commit/c881223f26ca784b8caed334d34ee3efbc8abbdb) 
   in working `Working with DEWP.md`.
   Remove local `list-woo-dewped` script.

### Screenshots:

![image](https://github.com/woocommerce/google-listings-and-ads/assets/17435/7a53e5f7-b4f3-4f04-a7e7-c9578e3d0b4b)



### Detailed test instructions:


1. Checkout branch
2. Run `npm run outdated:dewp`
3. Read the [`Working with DEWP.md`](https://github.com/woocommerce/google-listings-and-ads/compare/dev/dewped-docs?expand=1#diff-67f51433edb0d258d45ab8af9a0a99c32af2d1383c5aaa018a00a297fcb46d14) check if it's helpful, and suggested snippets work.


### Additional details:


### Changelog entry

> Dev - Update DEWP related tools and docs.
